### PR TITLE
ACQ-667 lp ai pl 101 fix iframe and logos postion

### PR DESCRIPTION
**Pull request by dmantonyuk**
## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

**Comment by dmantonyuk** [comment]
image

Dani:

**Comment by dmantonyuk** [comment]
Dami:
1) The form in the middle of the page is still not showing the right questions for me. It should have a question "What best describes your role in education?D" for example. Any idea what is going on there? ( I attached a screenshot below)

2) Can we put the partners in the order [Code.org](http://code.org/), ETS, ISTE, and Khan Academy through out the page? Let me know if it would be helpful for me to update the doc.

3) Are you seeing a scroll bar next to the form because I'm not seeing that.

Dmytro: 
1) form fixed but we have banner. Dami will work with Meghan and Pam to remove banner

2) logo are in right order

3) Scroll bar is visible only on window pc, so I increased height 
- after form herder banner will be removed we will need one more PR to remove bottom space 


cc: @TurnerRiley  @dmcavoy

File: pegasus/sites.v3/code.org/public/ai/pl/101/index.haml [hunk 1]
```diff
@@ -24,9 +24,9 @@ theme: responsive_full_width
         %span{style: "color: white"}=hoc_s(:ai_pl_101_hero_presented)
         .hero-ai-pl-webinar__list
           %img{src: "/images/ai/pl/101/code--w.png", height: "24px", alt: "CODE"}
-          %img{src: "/images/ai/pl/101/khan--w.png", height: "20px", alt: "Khan Academy"}
           %img{src: "/images/ai/pl/101/iste--w.png", height: "18px", alt: "ISTE"}
           %img{src: "/images/ai/pl/101/ets--w.png", height: "26px", alt: "ets"}
```
Comment (TurnerRiley) [inline pegasus/sites.v3/code.org/public/ai/pl/101/index.haml:28]: I believe the ETS should be before ISTE here

File: pegasus/sites.v3/code.org/public/ai/pl/101/index.haml [hunk 2]
```diff
@@ -96,6 +97,6 @@ theme: responsive_full_width
       %p=hoc_s(:ai_pl_101_partnership_between)
       .partnership-ai__list
         %img.partnership-ai__logo.partnership-ai__logo--code{src: "/images/ai/pl/101/code.svg", height: "50px", alt: "CODE"}
-        %img.partnership-ai__logo.partnership-ai__logo--khan{src: "/images/ai/pl/101/khan.svg", height: "50px", alt: "Khan Academy"}
         %img.partnership-ai__logo.partnership-ai__logo--iste{src: "/images/ai/pl/101/iste.png", height: "38px", alt: "ISTE"}
         %img.partnership-ai__logo.partnership-ai__logo--ets{src: "/images/ai/pl/101/ets.png", height: "55px", alt: "ETS"}
```
Comment (TurnerRiley) [inline pegasus/sites.v3/code.org/public/ai/pl/101/index.haml:101]: Same here, I believe the ETS should be before ISTE here

**Review by TurnerRiley — APPROVED** [review]
Just had a few comments, but looks good!

**Pull request merged**